### PR TITLE
TST: fix e2e tests

### PR DESCRIPTION
**Pull request by limpbrains**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/60w756367fg5hant9cv3c4t1hr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7742)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/1c46ruc3zhe3e19avpmqygxtj8?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7743)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/uwc7bk9chmwhbktn5wyu19u5jw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7746)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/h3ntqn4xbb52n4updukt1d55mg?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7748)

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.9-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7.1-all.zip
```
Comment (Overtorment) [inline android/gradle/wrapper/gradle-wrapper.properties:6]: hm, so gradle bump has nothing to do with it?

Comment (limpbrains) [reply android/gradle/wrapper/gradle-wrapper.properties:6]: Yes. And if you try to re-run tests that was working a couple of days ago, they are now failing.
Something happened with our `reactivecircus/android-emulator-runner@v2` test runner or something

**Comment by limpbrains** [comment]
@marcosrdz maybe you have some ideas? Take a look at the log https://github.com/BlueWallet/BlueWallet/runs/3314429727

**Comment by marcosrdz** [comment]
@limpbrains  seems that its looking for 30.8.2 but the sys image downloaded contains 30.7.3. Meaning the system image being downloaded has a version that is below the minimum expected.

**Comment by Overtorment** [comment]
We did bump minSdk for android or smth, no? Recently

**Comment by marcosrdz** [comment]
minSdk is 29 i believe. Targetsdk is 31 due to google august changes

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/0ad3zfq7du876th51ey9k78rzw?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7752)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2c00qpbmdw2q4rzmtfedfwt77w?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7753)

**Comment by Overtorment** [comment]
i tried to revert to 6.2.2 `252ed337` and it still failed with the same error message

https://github.com/BlueWallet/BlueWallet/pull/3590

**Comment by Overtorment** [comment]
https://github.com/ReactiveCircus/android-emulator-runner/issues/179

**Comment by Overtorment** [comment]
and i had thoughts on removing travisCI recently... good thing I didnt, at least we have one e2e working

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/02cw0r679dd1pj22j04x7zruvr?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/7758)

**Comment by Overtorment** [comment]
fixed automagically

**Pull request closed without merging**